.github/workflows/rosa-run-crossdc-func-tests.yml

```diff
@@ -40,37 +49,70 @@ jobs:
           key: ${{ runner.os }}-maven-wrapper-${{ hashFiles('**/maven-wrapper.properties') }}
           restore-keys: |
             ${{ runner.os }}-maven-wrapper-
+
       - name: Setup ROSA CLI
         uses: ./.github/actions/rosa-cli-setup
         with:
           aws-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-default-region: ${{ vars.AWS_DEFAULT_REGION }}
           rosa-token: ${{ secrets.ROSA_TOKEN }}
+
       - name: Login to OpenShift cluster A
         uses: ./.github/actions/oc-keycloak-login
         with:
           clusterName: ${{ inputs.clusterPrefix }}-a
-      - name: Get DC1 URLs
+
+      - name: Get DC1 Infinispan URLs
         shell: bash
         run: |
-          KEYCLOAK_DC1_URL=https://$(kubectl get routes -n "${{ env.PROJECT }}" aws-health-route -o jsonpath='{.spec.host}')
+          ISPN_DC1_URL=https://$(kubectl get routes -n "${{ env.PROJECT }}" -l app=infinispan-service-external -o jsonpath='{.items[*].spec.host}')
+          echo "ISPN_DC1_URL=$ISPN_DC1_URL" >> "$GITHUB_ENV"
+
+      - name: Get DC1 Active/Passive URLs
+        if: ${{ !inputs.activeActive }}
+        shell: bash
+        run: |
+          KEYCLOAK_DC1_URL=https://$(kubectl get routes -n "${{ env.PROJECT }}" accelerator-loadbalancer -o jsonpath='{.spec.host}')
```

Comment:
That's a good point. I was finding it annoying when having to manually retrieve these values, I'll add that :+1:

Reply:
In order for us to add a split-brain test, I think we need to make the testsuite aware of our K8s clients so that the GossipRouter Deployment can be scaled down. If we do this, I can just implement the URL retrieval logic in Java and so there's no need for it to be in task files or GH actions.

For example we could make it so that in order to run the tests the following properties are required:

```
-Dkubernetes.api-url.1=
-Dkubernetes.api-url.2=
-Dkubernetes.username=
-Dkubernetes.password=
-Ddeployment.namespace=
-Ddeployment.type=active-active|active-passive
```
Then based upon ^ it should be possible to retrieve the various Route53 and Accelerator URLs.